pubtools/pulplib/_impl/model/unit/erratum.py

```diff
@@ -374,6 +374,15 @@ class ErratumUnit(Unit):
     )
     """A list of package collections associated with the advisory."""
 
+    container_list = pulp_attrib(
+        type=list,
+        pulp_field="pulp_user_metadata.container_list",
```

Comment:
Can you please double-check if a flat list of string is really sufficient?

This comment https://url.corp.redhat.com/952c018 said that it should also contain arch info.

Reply:
yeah, it's actually slightly complicated structure